pkg/apis/kops/v1alpha1/componentconfig.go

```diff
@@ -542,8 +542,9 @@ type OpenstackLoadbalancerConfig struct {
 }
 
 type OpenstackBlockStorageConfig struct {
-	Version  *string `json:"bs-version,omitempty"`
-	IgnoreAZ *bool   `json:"ignore-volume-az,omitempty"`
+	Version    *string `json:"bs-version,omitempty"`
+	IgnoreAZ   *bool   `json:"ignore-volume-az,omitempty"`
+	OverrideAZ *string `json:"override-volume-az,omitempty"`
```

Comment:
Nit: probably less confusing for developers to name the field the same as the json, i.e. `OverrideVolumeAZ` in this case.